packages/flutter_tools/bin/tool_backend.dart

```diff
@@ -18,6 +18,9 @@ Future<void> main(List<String> arguments) async {
   final String localEngine = Platform.environment['LOCAL_ENGINE'];
   final String flutterRoot = Platform.environment['FLUTTER_ROOT'];
 
+  final String flutterExecutable = path.join(
```

Comment:
Move this after the localEngine check to avoid initializing and exiting early?

Reply:
Done